sections/02_intro.md

```diff
@@ -12,23 +12,22 @@ meaningful patterns and provide actionable knowledge allowing us to better
 treat, categorize, or study disease, all within data constrained and privacy
 critical environments.
 
-Concurrent with this explosive growth in biomedical data, a new class of machine
-learning algorithm - artificial neural networks, also known as deep learning -
-is revolutionizing domains from X to Y (I dunno - playing chess? serving ads?).
-As recently applied to image analysis problems,
-these architectures are blowing away prior best-in-class results, and
-computer scientists are now building many-layered neural networks from
-collections of millions of images. In an early famous example, scientists from
-Google demonstrated that a neural network could learn to identify cats simply by
-watching online videos
+Concurrent with this explosive growth in biomedical data, a new enthusiasm for a
+class of machine learning algorithms, known as deep learning, is revolutionizing
+domains from image search to the game of Go [@doi:10.1038/nature16961]. As
```

Comment:
Yeah, I was the one who requested the tags because it suits my own writing style better to have the human-readable form.  So far some sections have used them, others haven't.